content/en/tracing/trace_collection/trace_context_propagation/_index.md

```diff
@@ -11,8 +11,25 @@ further_reading:
     - link: 'https://www.datadoghq.com/blog/monitor-otel-with-w3c-trace-context/'
       tag: 'Blog'
       text: 'Monitor OpenTelemetry-instrumented apps with support for W3C Trace Context'
+    - link: '/opentelemetry/guide/otel_api_tracing_interoperability'
+      tag: 'Documentation'
+      text: 'Interoperability of OpenTelemetry API and Datadog instrumented traces'
 ---
 
+W3C trace propagation and 128 bit headers is available for all supported languages. The default trace propagation style for each language is:
+
+| Language | Default inject and extract setting |
+| -------- | ---------------------------------- |
+| Java     | `datadog,tracecontext`             |
+| Python   | `tracecontext,datadog`             |
+| Ruby     | `datadog,tracecontext`             |
+| Go       | `tracecontext,datadog`             |
+| Node.js  | `datadog,tracecontext`             |
+| PHP      | `tracecontext,datadog`             |
+| .NET     | `tracecontext,datadog`             |
+| C++      | `datadog,tracecontext` or with Envoy or nginx proxies: `tracecontext,Datadog` |
```

Comment:
Would it be okay for the Envoy and nginx proxies to have their own line in this table? cc @dgoffredo